chains/manager.go

```diff
@@ -258,21 +200,16 @@ type manager struct {
 	// Key: Chain's ID
 	// Value: The chain
 	chains map[ids.ID]handler.Handler
-
-	// snowman++ related interface to allow validators retrieval
-	validatorState validators.State
 }
 
 // New returns a new Manager
 func New(config *ManagerConfig) Manager {
 	return &manager{
-		Aliaser:                ids.NewAliaser(),
+		Aliaser:                config.Aliaser,
 		ManagerConfig:          *config,
-		stakingSigner:          config.StakingTLSCert.PrivateKey.(crypto.Signer),
-		stakingCert:            staking.CertificateFromX509(config.StakingTLSCert.Leaf),
 		chains:                 make(map[ids.ID]handler.Handler),
 		chainsQueue:            buffer.NewUnboundedBlockingDeque[ChainParameters](initialQueueSize),
-		unblockChainCreatorCh:  make(chan struct{}),
+		unblockChainCreatorCh:  config.UnblockChainCreatorCh,
```

Comment:
Do you mean as a parameter like `StartChainCreator(...)`? If so I think beyond this PR I would like for this channel to not even be in this struct... I feel like the caller of this code (node) should choose when this goroutine starts. I think in an ideal world we have some channel or event subscription thing where the subnet can publish a bootstrapped event and the node spins up goroutine that blocks until it sees the bootstrapped event. For now I left it as a field this struct instead of as a parameter since it keeps the current behavior.

If instead you meant just use `config.UnblockChainCreatorCh` instead of keeping this field then yeah that makes sense to me